compose/galaxy-server/Dockerfile

```diff
@@ -130,6 +130,10 @@ COPY --chown=$GALAXY_USER:$GALAXY_USER --from=build_miniconda /etc/profile.d/con
 
 COPY ./files/start.sh /usr/bin/start.sh
 
+RUN usermod -aG root $GALAXY_USER
```

Comment:
@pvanheus can we remove all those changes in this file? I think the Galaxy upgrade should happen in a different PR.

Reply:
That's true. Also there's stuff in there that needs to be made conditional - i.e. the ability of the Galaxy and IRIDA containers to talk to each other (share users).